cmd/mirror-main.go

```diff
@@ -637,7 +658,10 @@ func mainMirror(ctx *cli.Context) error {
 	srcURL := args[0]
 	tgtURL := args[1]
 
-	runMirror(srcURL, tgtURL, ctx)
+	if err := runMirror(srcURL, tgtURL, ctx); err != nil {
+		exitStatus(globalErrorExitStatus)
```

Comment:
yeah you are right, but even like that it works (it seems the next line is enough), I'll fix that to be more generic.